target/linux/ramips/image/mt76x8.mk

```diff
@@ -359,6 +359,17 @@ define Device/tplink_tl-wa801nd-v5
 endef
 TARGET_DEVICES += tplink_tl-wa801nd-v5
 
+define Device/tplink_tl-wa850re-v4
+  $(Device/tplink-v1)
+  DEVICE_MODEL := TL-WA850RE
+  DEVICE_VARIANT := v4
+  IMAGE_SIZE := 3712k
+  TPLINK_HWID := 0
```

Comment:
Reminder: This needs to be updated

Reply:
Based on my googling I assume 0x08500004 is okay and this is a value only used by openwrt for matching that the sysupgrade image is for the correct device?

I thought it was something fundamental to the hardware and mktplinkfw -i on the vendor fw only reported a zero...

Reply:
Sysupgrade uses SUPPORTED_DEVICES. This should be required for the WebUI to accept it (if flashing from WebUI via factory image is possible for this device at all.)